app/core/models/ocelot/LabelCache.scala

```diff
@@ -20,31 +20,55 @@ import play.api.i18n.Lang
 
 trait Labels {
   def value(name: String): Option[String]
+  def valueAsList(name: String): Option[List[String]]
   def displayValue(name: String)(implicit lang: Lang): Option[String]
   def update(name: String, english: String): Labels
   def update(name: String, english: String, welsh: String): Labels
+  def updateList(name: String, english: List[String]): Labels
+  def updateList(name: String, english: List[String], welsh: List[String]): Labels
   def updatedLabels: Map[String, Label]
   def labelMap:Map[String, Label]
   def flush(): Labels
 }
 
 private class LabelCacheImpl(labels: Map[String, Label] = Map(), cache: Map[String, Label] = Map()) extends Labels {
-  def value(name: String): Option[String] = label(name).map(_.english.getOrElse(""))
+  def value(name: String): Option[String] = label(name).fold[Option[String]](None)
+    {case s:ScalarLabel => Some(if(s.english.nonEmpty)s.english.head else "") case _:ListLabel => None}
```

Comment:
can be simplified using collect() 
e.g. `label(name).collect{case s:ScalarLabel => s.english.headOption.getOrElse("")}`

Reply:
Done